server/controllers/user.js

```diff
@@ -40,11 +41,61 @@ export const userLogin = async (req, res) => {
       data: null
     });
   } catch (err) {
+    return serverError(res, err);
+  }
+};
+
+export const passwordResetRequest = async (req, res) => {
+  const { email } = req.body;
+  try {
+    const verifiedUser = await User.findOne({
+      where: {
+        email
+      }
+    });
```

Comment:
Does email alone indicate that a user is `verified`. You need to include a check that the user is a verified user.
```
const verifiedUser = await User.findOne({
  where: {
    email,
    isVerified: true
  }
})
```